src/sentry/incidents/grouptype.py

```diff
@@ -63,14 +71,30 @@ def create_occurrence(
             raise DetectorException(
                 f"Failed to find snuba query for detector id {self.detector.id}, cannot create metric issue occurrence"
             )
-        occurrence = DetectorOccurrence(
-            issue_title=self.construct_title(snuba_query, detector_trigger, priority),
-            subtitle="An Issue Subtitle",
-            type=MetricIssue,
-            level="error",
-            culprit="Some culprit",
+
+        try:
+            assignee = parse_and_validate_actor(
+                str(self.detector.created_by_id), self.detector.project.organization_id
+            )
+        except Exception:
+            assignee = None
+
+        return (
+            DetectorOccurrence(
+                issue_title=self.detector.name,
+                subtitle=self.construct_title(snuba_query, detector_trigger, priority),
+                evidence_data={
+                    "alert_id": alert_id,
+                },
+                evidence_display=[],  # XXX: may need to pass more info here for the front end
+                type=MetricIssue,
+                level="error",
+                culprit="",
+                assignee=assignee,
+                priority=priority,
+            ),
+            {},
```

Comment:
In the previous attempt I was setting some event data: https://github.com/getsentry/sentry/blob/8303446117a6231e72b4712f9f3772cabf1c10e7/src/sentry/incidents/grouptype.py#L83-L87

it seems that having a "platform" of None was the main problem and the other ones either aren't necessary or are covered by the base class